src/components/SignInButtons/AppleSignIn/index.desktop.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {View} from 'react-native';
+import withLocalize, {withLocalizePropTypes} from '../../withLocalize';
+import ButtonBase from '../ButtonBase';
+import AppleLogoIcon from '../../../../assets/images/signIn/apple-logo.svg';
+import CONFIG from '../../../CONFIG';
+import ROUTES from '../../../ROUTES';
+
+const propTypes = {...withLocalizePropTypes};
+
+const $appleButtonContainerStyle = {
+    width: 40,
+    height: 40,
+    marginRight: 20,
+};
+
+//
+const appleSignInWebRouteForDesktopFlow = `${CONFIG.EXPENSIFY.NEW_EXPENSIFY_URL}${ROUTES.APPLE_SIGN_IN}`;
+
+const AppleSignIn = (props) => (
+    <View
+        style={$appleButtonContainerStyle}
+        accessibilityRole="button"
+        accessibilityLabel={props.translate('common.signInWithApple')}
+    >
```

Comment:
I think we might want to try moving these into button base, and passing apple/google specific data as props